### PR TITLE
Create a section about javascript debugging

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: best-practices/javascript_debugging.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Debugging in Javascript
+
+If you'd like to use a stepwise debugging in javascript, consider using the following procedure when working in VSCode.
```
Comment (aaron-collier) [inline best-practices/javascript_debugging.md:3]: Should this read "If you'd like use stepwise..."? Other thank, :+1:

**Review by aaron-collier — APPROVED** [review]

File: best-practices/javascript_debugging.md (hunk 1 above)
Comment (jcoyne) [reply best-practices/javascript_debugging.md:3]:
```suggestion
If you'd like to use stepwise debugging in javascript, consider using the following procedure when working in VSCode.
```

**Review by justinlittman — APPROVED** [review]

File: best-practices/javascript_debugging.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Debugging in Javascript
+
+If you'd like to use stepwise debugging in javascript, consider using the following procedure when working in VSCode.
+
+* Type <CMD><Shift>p to bring up the command palette
```
Comment (jmartin-sul) [inline best-practices/javascript_debugging.md:5]:
```suggestion
* Type `<CMD><Shift>p` to bring up the command palette
```

looks like markdown tries to render the parts in angle brackets as tags (if you "view file" on this).  hopefully this suggested edit takes care of that?

File: best-practices/javascript_debugging.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+* Search for “Debug: JavaScript Debug Terminal”
+* In the terminal enter your test invocation, e.g. npx jest __test__/components/Whatever.test.js
```
Comment (jmartin-sul) [inline best-practices/javascript_debugging.md:7]:
```suggestion
* In the terminal enter your test invocation, e.g. `npx jest __test__/components/Whatever.test.js`
```

and then MD wants to render `__test__` as bold if there are no formatting backticks.

**Review by jmartin-sul — APPROVED** [review]
a couple formatting suggestions, but approving because the instructions are still useful as-is.

**Pull request merged**